.sbtopts-nightly-sigar

```diff
@@ -0,0 +1,19 @@
+-J-XX:+UseG1GC -Xms3g -Xmx3g 
```

Comment:
let's keep the jvm settings in one place https://github.com/akka/akka/blob/master/.jvmopts-ci
and is fine to remove:
```
-J-XX:+PrintGCDetails 
-J-XX:+PrintGCTimeStamps 
```

Reply:
They are different so will add a `.jvmopts-nightly-cluster-metrics-sigar` and copy it.

Reply:
Each single job has a completely different set of arguments.

Reply:
let's unify them as much as possible

Reply:
I'll try to run the standard `.jvmopts-ci`